packages/core/src/FlyoutMenu/features/position/index.js

```diff
@@ -1,5 +1,7 @@
 import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps'
 
+const CLOSE_TO_DELTA = 1
```

Comment:
Did you figure out what was causing the differences compared to the earlier tests (pre storybook upgrade)?

Reply:
No

Reply:
Ok. I guess it's not really that big of a deal as it's just a 1px delta.

Reply:
yeah I guess we can attribute that to rounding issues and it doesn't matter too much